main.js

```diff
@@ -67,37 +68,37 @@ function randomize() {
   var randomMessage = findType(affirmationButton, mantraButton)[randomIndex];
   displayMessage(randomMessage);
 }
-function displayMessage(message) {
+function hideBellShowButtons(){
   bellImage.classList.add('hidden');
-  deletedMessageOutput.classList.add('hidden');
-  addMessageForm.classList.add('hidden');
-  messageOutput.innerText = message;
   clearMessageButton.classList.remove('hidden');
   deleteMessageButton.classList.remove('hidden');
-
 }
-function clearMessage() {
+function showDefaultView() {
   addMessageForm.classList.add('hidden');
   messageOutput.innerText = "";
   clearMessageButton.classList.add('hidden');
   deleteMessageButton.classList.add('hidden');
   bellImage.classList.remove('hidden');
 }
+function displayMessage(message) {
+  deletedMessageOutput.classList.add('hidden');
+  addMessageForm.classList.add('hidden');
+  messageOutput.innerText = message;
+  hideBellShowButtons();
+}
 function deleteMessage() {
   if (confirm('Are you sure you want to delete this message? It will no longer be available.')) {
-    for (var i = 0; i < findType(affirmationButton, mantraButton).length; i++) {
-      if (findType(affirmationButton, mantraButton)[i] === messageOutput.innerText) {
-        var deletedMessage = findType(affirmationButton, mantraButton)[i];
-        findType(affirmationButton, mantraButton).splice(i, 1);
+    for (var i = 0; i < messageTypes.length; i++) {
+      for (var j = 0; j < messageTypes[i].length; j++) {
+        if (messageTypes[i][j] === messageOutput.innerText) {
+          var deletedMessage = messageTypes[i][j];
+          messageTypes[i].splice(j, 1);
+        }
```

Comment:
Also updated this for loop to loop through both arrays to find the message, no matter what radio button is selected at the time.